### DISCUSSION TITLE
DAI 2023-2024 - Practical work 1 - Pirakas Anthon

### DISCUSSION THREAD

**Discussion by apirakas** [heig-vd-dai-course/heig-vd-dai-course#94] [Show and tell]
I'm going to do the same cli that was shown during the presentation and I'm going to add some new features like capitalizing a defined word in the text file.

repo of my code : https://github.com/apirakas/CaseSwapCli/tree/431e70169d8e3ea898fb800597b78e8468644183

**Comment by ludelafo** [comment]
Great! Do you already have a list of all the new features you would like to add or it is not defined yet?

**Reply by apirakas** [reply]
For the moment, I've already added this feature in local, which consists in transforming (into lower or upper case) a word defined in the text file. 

It works like this: the word is transformed in the output file and then the console shows how many times it has changed the word.

Is this enough or do I need to add another new option ?

**Reply by ludelafo** [reply]
I think this is enough. Maybe you could add an optional output file to save the number of times the word was changed as well so you can always keep a track of the changes!
